surveys/templates/partials/sidebar.html

```diff
@@ -71,7 +74,11 @@ <h3><b>Just Spaces</b></h3>
             <div><a class="btn-sidebar" href="{% url 'signup' %}">Create new account</a></div>
             {% endif %}
             <br />
-            <div class="about-blurb"><strong>Just Spaces</strong> is a tool from <a class="about-link" href="https://www.universitycity.org/" target=_blank>University City District</a> and <a class="about-link" href="https://datamade.us/" target=_blank>DataMade</a> to promote better and more just public spaces</div>
+            <div class="about-blurb">
+              <strong>Just Spaces</strong> is a tool from <a class="about-link" href="https://www.universitycity.org/" target=_blank>University City District</a> to promote better and more just public spaces
+            </div>
+            <br />
+            <div>Built by <a class="about-link" href="https://datamade.us/" target=_blank>DataMade</a></div>
```

Comment:
Our typical footer text is `Website by DataMade [and partner]`, see e.g. https://largelots.org/.

Reply:
Derek signed off on `Built by`, so I'm resolving this.